src/graphql/blocks.ts

```diff
@@ -1,13 +1,36 @@
 import snapshot from '@snapshot-labs/snapshot.js';
 import networks from '@snapshot-labs/snapshot.js/src/networks.json';
 import redis from '../redis';
+import fetch from 'cross-fetch';
 
 const starts: any = {};
 Object.keys(networks).forEach((network) => {
   starts[network] = networks[network].start || 1;
 });
 
-async function tsToBlockNum(network, ts) {
+export async function tsToBlockWithApi(network: string, ts: number) {
+  const explorerBaseUrl = networks[network].explorer.apiUrl ?? networks[network].explorer.url;
+  const url = `${explorerBaseUrl}/api?module=block&action=getblocknobytime&timestamp=${ts}&closest=after`;
+
+  console.log(url);
```

Comment:
We shouldn't have this console log here
```suggestion
  console.log(url);
```